lms/djangoapps/philu_api/views.py

```diff
@@ -254,3 +256,17 @@ def send_alquity_fake_confirmation_email(request):
         logging.exception(ex)
         success = False
     return JsonResponse({'success': success})
+
+
+def resend_activation_email(request):
+    user = request.user
+    activated = user.is_active
+    try:
+        if not activated:
+            reactivation_email_for_user_custom(request, user)
+            return JsonResponse({'success': True, 'email': user.email}, status=status.HTTP_200_OK)
+        else:
+            return JsonResponse({'success': False}, status=status.HTTP_409_CONFLICT)
```

Comment:
Please specify the reason behind "409 Conflict"

Reply:
Done